manuscript/050_Its_Not_a_Test.md

```diff
@@ -68,12 +68,12 @@ The time has come to make a deal: I will establish some naming conventions for
 
 :   will be used instead of ‘passing test’
 
-From time to time, I will refer back to the ‘traditional’ terminology, because it is better established and you’ve probably heard some terms already and may wonder how it should be understood in context of thinking of tests as specification.
+From time to time I'll refer back to the ‘traditional’ terminology, because it is better established and because you’ve probably already heard some terms and may wonder how it should be understood in the context of thinking of tests as specification.
 
-From your experience, you may know paper or word specifications, written in plain English or other spoken language. Our specification is different than these specifications in at least few ways:
+From your experience, you may know paper or word specifications written in plain English or other spoken language. Our specification is different from these specifications in at least few ways:
```

Comment:
You are right that by this sentence, I meant old-fashioned specifications, though I think "old-fashioned" is a bit ambiguous. I really meant requirements or design specifications that were usually considered a precondition to start implementing something.